build-cran-package.sh

```diff
@@ -91,10 +91,10 @@ mkdir -p "${EIGEN_R_DIR}"
 
 modules="Cholesky Core Dense Eigenvalues Geometry Householder Jacobi LU QR SVD"
 for eigen_module in ${modules}; do
-    cp external_libs/eigen/Eigen/${eigen_module} "${EIGEN_R_DIR}/${eigen_module}"
-    if [ ${eigen_module} != "Dense" ]; then
+    cp external_libs/eigen/Eigen/"${eigen_module}" "${EIGEN_R_DIR}/${eigen_module}"
```

Comment:
```suggestion
    cp "external_libs/eigen/Eigen/${eigen_module}" "${EIGEN_R_DIR}/${eigen_module}"
```

Can you please wrap entire paths in `"` instead of inserting the `"` in the middle of them? I think that's a little easier to read (personal preference).